packages/functional-tests/pages/login.ts

```diff
@@ -378,22 +386,58 @@ export class LoginPage extends BaseLayout {
     return this.page.locator(selectors.SUBMIT).click();
   }
 
-  async waitForCWTSHeader() {
-    await this.page.waitForSelector(selectors.CWTS_HEADER, {
-      timeout: 100,
+  async waitForCWTSEngineHeader() {
+    await this.page.waitForSelector(selectors.CWTS_ENGINE_HEADER, {
+      timeout: 2000,
     });
   }
 
-  async waitForCWTSEngineBookmarks() {
-    await this.page.waitForSelector(selectors.CWTS_ENGINE_BOOKMARKS, {
-      timeout: 100,
-    });
+  async isCWTSPageHeader() {
+    return this.page.locator(selectors.CWTS_PAGE_HEADER).isVisible();
   }
 
-  async waitForCWTSEngineHistory() {
-    await this.page.waitForSelector(selectors.CWTS_ENGINE_HISTORY, {
-      timeout: 100,
-    });
+  async isCWTSEngineCreditCards() {
+    return this.page.locator(selectors.CWTS_ENGINE_CREDITCARDS).isVisible();
+  }
+
+  async isCWTSEngineBookmarks() {
+    return this.page.locator(selectors.CWTS_ENGINE_BOOKMARKS).isVisible();
+  }
+
+  async isCWTSEngineHistory() {
+    return this.page.locator(selectors.CWTS_ENGINE_HISTORY).isVisible();
+  }
+
+  async isCWTSEnginePassword() {
+    return this.page.locator(selectors.CWTS_ENGINE_PASSWORDS).isVisible();
+  }
+
+  async isCWTSEngineAddons() {
+    return this.page.locator(selectors.CWTS_ENGINE_ADDONS).isVisible();
+  }
+
+  async isCWTSEnginePrefs() {
+    return this.page.locator(selectors.CWTS_ENGINE_PREFS).isVisible();
+  }
+
+  async isCWTSEngineTabs() {
+    return this.page.locator(selectors.CWTS_ENGINE_TABS).isVisible();
+  }
+
+  async isCWTSEngineAddresses() {
+    return this.page.locator(selectors.CWTS_ENGINE_ADDRESSES).isVisible();
+  }
+
+  async isDoNotSync() {
+    return this.page.locator(selectors.DO_NOT_SYNC).isVisible();
+  }
+
+  async uncheckCWTSEngineHistory() {
+    await this.page.locator(selectors.CWTS_ENGINE_HISTORY).click();
+  }
+
+  async uncheckCWTSEnginePasswords() {
+    await this.page.locator(selectors.CWTS_ENGINE_PASSWORDS).click();
   }
```

Comment:
nit: not sure if the `uncheck*` prefix is misleading since this toggles the current value (either on or off).
Should we rename these to `toggle*`, or possibly irrelevant since we never click a checkbox more than once.

Reply:
I will make a ticket for this in the cleanup epic here (https://mozilla-hub.atlassian.net/browse/FXA-7900)